grakn-test/test-integration/src/test/java/ai/grakn/test/migration/owl/TestSubProperties.java

```diff
@@ -35,16 +35,16 @@
 import static ai.grakn.graql.Graql.var;
 
 public class TestSubProperties extends TestOwlGraknBase {
-	private final IRI baseIri = IRI.create("http://www.workingontologist.org/Examples/Chapter3/shakespeare.owl");
```

Comment:
@fppt did we just unearth an ancient OWL migrator test?

Reply:
Looks like it. Kill!